apps/civicsignalblog/src/payload/collections/Users.js

```diff
@@ -1,3 +1,6 @@
+import payload from "payload";
+
+import applications from "../../lib/data/json/applications";
```

Comment:
Not sure I fully understand the naming of "managed applications"? Who is managing them and why are they managing them?

I think authorised applications/apps or just applications/apps would make a lot more sense (unless there is a reason behind the "managed" part)

Reply:
I kept "managed" since they are apps whose content is being managed by our CMS 😄 ... but come to think of it, its somewhat redundant...I think we can keep it as just "app"